llvm/test/Transforms/SimplifyCFG/X86/switch_to_lookup_table_big.ll

```diff
@@ -0,0 +1,534 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt < %s -passes=simplifycfg -switch-to-lookup=true -keep-loops=false -S | FileCheck %s
```

Comment:
I don't think `-keep-loops` is relevant here? We also don't need both RUN lines, they do the same thing.

Reply:
Alright, I've removed `-keep-loops=false` and the second `RUN`. I copied those lines from the original `switch_to_lookup_table.ll`; I assumed there was some reason why both versions were being tested.